iotdb/src/main/java/org/apache/iotdb/db/engine/overflow/io/OverflowResource.java

```diff
@@ -184,9 +186,10 @@ public void flush(FileSchema fileSchema, IMemTable memTable,
     timeInterval = timeInterval == 0 ? 1 : timeInterval;
     long insertSize = insertIO.getPos() - startPos;
     LOGGER.info(
-        "Overflow processor {} flushes overflow insert data, actual:{}, time consumption:{} ms, flush rate:{}/s",
-        processorName, MemUtils.bytesCntToStr(insertSize), timeInterval,
-        MemUtils.bytesCntToStr(insertSize / timeInterval * 1000));
+            "Overflow processor {} flushes overflow insert data, actual:{}, time consumption:{} ms,"
+                    + " flush rate:{}/s",
+            processorName, MemUtils.bytesCntToStr(insertSize), timeInterval,
```

Comment:
wrap this log with if(LOGGER.isInfoEnabled()) is better.

Reply:
Fixed.